data/json/items/comestibles/egg.json

```diff
@@ -0,0 +1,201 @@
+    "copy-from": "egg_bird",
+    "rot_spawn": "GROUP_EGG_CHICKEN",
+    "rot_spawn_chance": 70
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_grouse",
+    "name": "bird egg",
+    "copy-from": "egg_chicken",
+    "rot_spawn": "GROUP_EGG_GROUSE"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_crow",
+    "name": "bird egg",
+    "copy-from": "egg_chicken",
+    "rot_spawn": "GROUP_EGG_CROW"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_duck",
+    "name": "bird egg",
+    "copy-from": "egg_chicken",
+    "rot_spawn": "GROUP_EGG_DUCK"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_goose_canadian",
+    "name": "bird egg",
+    "copy-from": "egg_chicken",
+    "rot_spawn": "GROUP_EGG_GOOSE_CANADIAN"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_turkey",
+    "name": "bird egg",
+    "copy-from": "egg_chicken",
+    "rot_spawn": "GROUP_EGG_TURKEY"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_pheasant",
+    "name": "bird egg",
+    "copy-from": "egg_chicken",
+    "rot_spawn": "GROUP_EGG_PHEASANT"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_cockatrice",
+    "name": "bird egg",
+    "copy-from": "egg_chicken",
+    "rot_spawn": "GROUP_EGG_COCKATRICE"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_reptile",
+    "name": "reptile egg",
+    "weight": 60,
+    "color": "white",
+    "spoils_in": 672,
+    "comestible_type": "FOOD",
+    "symbol": "o",
+    "quench": 3,
+    "healthy": 1,
+    "nutrition": 10,
+    "description": "An egg belonging to one of reptile species found in New England.",
+    "price": 250,
+    "material": "egg",
+    "volume": 1,
+    "stack_size": 4,
+    "fun": -12,
+    "rot_spawn": "GROUP_EGG_SNAKE",
+    "rot_spawn_chance": 60
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "ant_egg",
+    "name": "ant egg",
+    "weight": 220,
+    "color": "white",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "spoils_in": 168,
+    "quench": 10,
+    "nutrition": 200,
+    "description": "A large white ant egg, the size of a softball.  Extremely nutritious, but incredibly gross.",
+    "price": 80,
+    "material": "egg",
+    "volume": 1,
+    "fun": -10,
+    "rot_spawn": "GROUP_EGG_ANT",
+    "rot_spawn_chance": 60
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "spider_egg",
+    "name": "spider egg",
+    "weight": 220,
+    "color": "white",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": 10,
+    "nutrition": 20,
+    "description": "A fist sized egg from from a giant spider.  Incredibly gross.",
+    "price": 80,
+    "material": "egg",
+    "volume": 1,
+    "fun": -20
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_roach",
+    "name": "roach egg",
+    "weight": 60,
+    "color": "white",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "spoils_in": 112,
+    "healthy": -1,
+    "quench": 3,
+    "nutrition": 8,
+    "description": "A fist sized egg from from a giant roach.  Incredibly gross.",
```

Comment:
Recommend correcting to: `A fist-sized egg from a giant roach.  Incredibly gross.`